docs/topics/installing-intellij-idea-plugin.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-extension_{context}"]
+= Installing the {ProductShortName} extension for IntelliJ IDEA
+
+You can install the {ProductShortName} extension in the Ultimate and the Community Edition releases of IntelliJ IDEA.
+
+.Prerequisites
+include::snippet_jdk-hardware-mac-prerequisites.adoc[]
+
+* The latest version of `mta-cli` from the link:{MTADownloadPageURL}[{ProductShortName} download page]
+
+.Procedure
+
+. In IntelliJ IDEA, click *Plugins* on the Activity bar.
+. Enter `{ProductName}` in the Search field of the *Marketplace* tab.
+. Select the *Migration Toolkit for Applications (MTA) by Red Hat* plugin and click *Install*.
++
+The plugin is listed in the *Installed* tab.
```

Comment:
[ISG](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#tab):  "on the ... tab".